packages/nodes-base/nodes/Todoist/v2/TodoistV2.node.ts

```diff
@@ -304,6 +549,61 @@ const versionDescription: INodeTypeDescription = {
 			required: true,
 			description: 'Task content',
 		},
+		{
+			displayName: 'Text',
+			name: 'text',
+			type: 'string',
+			typeOptions: {
+				rows: 3,
+			},
+			displayOptions: {
+				show: {
+					resource: ['task'],
+					operation: ['quickAdd'],
+				},
+			},
+			default: '',
+			required: true,
+			description:
+				'Natural language text for quick adding task (e.g., "Buy milk @Grocery #shopping tomorrow"). It can include a due date in free form text, a project name starting with the "#" character (without spaces), a label starting with the "@" character, an assignee starting with the "+" character, a priority (e.g., p1), a deadline between "{}" (e.g. {in 3 days}), or a description starting from "//" until the end of the text.',
+		},
+		{
+			displayName: 'Additional Fields',
+			name: 'options',
+			type: 'collection',
+			placeholder: 'Add Field',
+			default: {},
+			displayOptions: {
+				show: {
+					resource: ['task'],
+					operation: ['quickAdd'],
+				},
+			},
+			options: [
+				{
+					displayName: 'Note',
+					name: 'note',
+					type: 'string',
+					default: '',
+					description: 'The content of the note',
+				},
+				{
+					displayName: 'Reminder',
```

Comment:
this one is weird. adding reminder does not always work for me. But works sometimes. Not sure if it's a limitation on my account or an issue with the API. Will leave it since it kinda works. 

I created changes with a "Reminders" resource, but stashed them for now. Can create a separate PR for that once we understand why it does not work consistently.